Add user interface for graph editor graph layout parameters (#406)
hexatomic/hexatomic: docs/user/src/usage/graph-editor/README.md

![Screenshot of the graph editor](graph-viewer.png)

> On the bottom of the graph editor, there is the console which you can use to actually edit the graph.

Reviewer: ```suggestion
On the bottom of the graph editor is the console that you can use to edit the graph.
```

Simply sentence.

- You can **center the view** around a specific point in the graph by double-clicking that point.

> If you don't like the layout of the graph, you can change it by simply dragging nodes with your mouse or [adjust the layout parameters](./layout-params.md).

Reviewer: ```suggestion
If you don't like the layout of the graph, you can change it by dragging nodes with your mouse or [adjusting the layout parameters](./layout-params.md).
```

Remove documentation smell, fix grammar.
